<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Info</class>
 <widget class="QWidget" name="Info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Info</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <attribute name="title">
       <string>Now Playing</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="QToolBar" name="col_toolbar">
         <addaction name="separator"/>
         <addaction name="actionBan"/>
         <addaction name="actionRateDown"/>
         <addaction name="actionRateUp"/>
         <addaction name="actionLoveIt"/>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>315</width>
            <height>422</height>
           </rect>
          </property>
          <layout class="QHBoxLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>1</width>
               <height>1</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QFrame" name="frame_0">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QFrame" name="frame_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="margin">
                  <number>4</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_so1">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>Song Title</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="StarEditor" name="songRating" native="true"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_al0">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>128</width>
                     <height>128</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>128</width>
                     <height>128</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="pixmap">
                    <pixmap resource="resource.qrc">:/icons/def_album.png</pixmap>
                   </property>
                   <property name="scaledContents">
                    <bool>true</bool>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_5">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="margin">
                  <number>4</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_al1">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>Album Title</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="StarEditor" name="albumRating" native="true"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_4">
                <property name="visible">
                 <bool>false</bool>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_ar0">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>128</width>
                     <height>128</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>128</width>
                     <height>128</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="pixmap">
                    <pixmap resource="resource.qrc">:/icons/def_album.png</pixmap>
                   </property>
                   <property name="scaledContents">
                    <bool>true</bool>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame">
                <property name="visible">
                 <bool>false</bool>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="margin">
                  <number>4</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_ar1">
                   <property name="text">
                    <string>Artist Name</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="StarEditor" name="artistRating" native="true"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::MinimumExpanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>1</width>
                  <height>1</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>1</width>
               <height>1</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QToolBar" name="col_toolbar2">
         <addaction name="separator"/>
         <addaction name="actionShowArtistInfo"/>
         <addaction name="actionShowAlbumInfo"/>
         <addaction name="actionShowLyrics"/>
         <addaction name="actionGet_Images"/>
        </widget>
       </item>
      </layout>
      <addaction name="actionShow_Song"/>
      <addaction name="actionShow_Album_Image"/>
      <addaction name="actionShow_Album_Title"/>
      <addaction name="actionShow_Artist_Image"/>
      <addaction name="actionShow_Artist_Name"/>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Artist</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="textEdit">
         <property name="autoFormatting">
          <set>QTextEdit::AutoAll</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Album</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="textEdit_2">
         <property name="autoFormatting">
          <set>QTextEdit::AutoAll</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionRateUp">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/star.png</normaloff>:/icons/star.png</iconset>
   </property>
   <property name="text">
    <string>Rate Up</string>
   </property>
   <property name="toolTip">
    <string>Rate Up</string>
   </property>
  </action>
  <action name="actionRateDown">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/antistar.png</normaloff>:/icons/antistar.png</iconset>
   </property>
   <property name="text">
    <string>Rate Down</string>
   </property>
   <property name="toolTip">
    <string>Rate Down</string>
   </property>
  </action>
  <action name="actionLoveIt">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/heart.png</normaloff>:/icons/heart.png</iconset>
   </property>
   <property name="text">
    <string>Love it</string>
   </property>
  </action>
  <action name="actionBan">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/ban.png</normaloff>:/icons/ban.png</iconset>
   </property>
   <property name="text">
    <string>Ban</string>
   </property>
  </action>
  <action name="actionShowArtistInfo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/def_artist.png</normaloff>:/icons/def_artist.png</iconset>
   </property>
   <property name="text">
    <string>Show Artist Info</string>
   </property>
  </action>
  <action name="actionShowAlbumInfo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/def_album.png</normaloff>:/icons/def_album.png</iconset>
   </property>
   <property name="text">
    <string>Show Album Info</string>
   </property>
  </action>
  <action name="actionShowLyrics">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/def_song.png</normaloff>:/icons/def_song.png</iconset>
   </property>
   <property name="text">
    <string>Show Lyrics</string>
   </property>
  </action>
  <action name="actionShow_Song">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Song</string>
   </property>
  </action>
  <action name="actionShow_Album_Image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Album Image</string>
   </property>
  </action>
  <action name="actionShow_Album_Title">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Album Title</string>
   </property>
  </action>
  <action name="actionShow_Artist_Image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show Artist Image</string>
   </property>
  </action>
  <action name="actionShow_Artist_Name">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Artist Name</string>
   </property>
  </action>
  <action name="actionGet_Images">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/download_art.png</normaloff>:/icons/download_art.png</iconset>
   </property>
   <property name="text">
    <string>Get Images</string>
   </property>
   <property name="toolTip">
    <string>Get images from Last.FM</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StarEditor</class>
   <extends>QWidget</extends>
   <header>stareditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>Info</receiver>
   <slot>tabChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBan</sender>
   <signal>triggered()</signal>
   <receiver>Info</receiver>
   <slot>slot_ban()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoveIt</sender>
   <signal>triggered()</signal>
   <receiver>Info</receiver>
   <slot>slot_loveIt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRateDown</sender>
   <signal>triggered()</signal>
   <receiver>Info</receiver>
   <slot>slot_rateDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRateUp</sender>
   <signal>triggered()</signal>
   <receiver>Info</receiver>
   <slot>slot_rateUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowArtistInfo</sender>
   <signal>triggered()</signal>
   <receiver>Info</receiver>
   <slot>showArtist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowAlbumInfo</sender>
   <signal>triggered()</signal>
   <receiver>Info</receiver>
   <slot>showAlbum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowLyrics</sender>
   <signal>triggered()</signal>
   <receiver>Info</receiver>
   <slot>showLyric()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_Song</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame_3</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_Album_Image</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_Album_Title</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame_5</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_Artist_Image</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame_4</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_Artist_Name</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGet_Images</sender>
   <signal>triggered()</signal>
   <receiver>Info</receiver>
   <slot>getImages()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>getImages()</slot>
 </slots>
</ui>
